core/storage/voiceover/gae_models_test.py

```diff
@@ -0,0 +1,136 @@
+
+    def test_get_voiceover_model_returns_correctly(self) -> None:
+        dummy_manual_voiceover_dict: voiceover_models.VoiceoverDict = {
+            'filename': 'filename1.mp3',
+            'file_size_bytes': 3000,
+            'needs_update': False,
+            'duration_secs': 6.1
+        }
+        dummy_autogenerated_voiceover_dict: voiceover_models.VoiceoverDict = {
+            'filename': 'filename2.mp3',
+            'file_size_bytes': 3500,
+            'needs_update': False,
+            'duration_secs': 5.9
+        }
+        voiceover_models.EntityVoiceoversModel.create_new(
+            feconf.ENTITY_TYPE_EXPLORATION, 'exp_id', 1, 'en-US', {
+                'content_0': {
+                    'manual': dummy_manual_voiceover_dict,
+                    'auto': dummy_autogenerated_voiceover_dict
+                    }
+                }
+        ).put()
+
+        entity_voiceovers_model = (
+            voiceover_models.EntityVoiceoversModel.get_model(
+                entity_type=feconf.ENTITY_TYPE_EXPLORATION,
+                entity_id='exp_id',
+                entity_version=1,
+                language_accent_code='en-US'))
+
+        self.assertEqual(entity_voiceovers_model.entity_type, 'exploration')
+        self.assertEqual(entity_voiceovers_model.entity_id, 'exp_id')
+        self.assertEqual(entity_voiceovers_model.entity_version, 1)
+        self.assertEqual(entity_voiceovers_model.language_accent_code, 'en-US')
+        voiceovers = entity_voiceovers_model.voiceovers
+        self.assertEqual(
+            voiceovers['content_0']['manual'], dummy_manual_voiceover_dict)
+        self.assertEqual(
+            voiceovers['content_0']['auto'], dummy_autogenerated_voiceover_dict)
+
+    def test_get_export_policy_not_applicable(self) -> None:
+        self.assertEqual(
+            voiceover_models.EntityVoiceoversModel.get_export_policy(),
+            {
+                'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'entity_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'entity_type': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'entity_version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'language_accent_code':
+                    base_models.EXPORT_POLICY.NOT_APPLICABLE,
+                'voiceovers': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            }
+        )
+
+    def test_get_deletion_policy_not_applicable(self) -> None:
+        self.assertEqual(
+            voiceover_models.EntityVoiceoversModel.get_deletion_policy(),
+            base_models.DELETION_POLICY.NOT_APPLICABLE)
+
+    def test_get_model_association_to_user_not_corresponding_to_user(
+        self
+    ) -> None:
+        model = voiceover_models.EntityVoiceoversModel
```

Comment:
@Nik-09 One note, you should name this model_class or model_cls, otherwise it can be confused with the model instance.

Reply:
Done.